### PR TITLE
fixing typo

### PR DESCRIPTION
**Pull request by dret**

### CODE REVIEW

File: specification/index.md [hunk 1]
```diff
@@ -17,5 +17,5 @@ The following pages introduce the syntax and structure of an OpenAPI document, i
 - [Content of Message Bodies](content): `content` and `schema`.
 - [Parameters and Payload of an Operation](parameters): `parameters` and `requestBody`.
 - [Reusing Descriptions](components): `components` and `$ref`.
-- [Providing Documentation and Examples](docs): `example` and `examples`.
+- [Providing Documentation and Examples](docs): `description` and `examples`.
```
Comment (lornajane) [inline specification/index.md:20]: I'm not sure this was a typo. `example` and `examples` are two different things.

**Comment by dret** [comment]
On 2023-09-26 12:07, Lorna Jane Mitchell wrote:
> I'm not sure this was a typo. |example| and |examples| are two different 
> things.

fair point but the section seems to be about `description` and 
`example`/`examples`.

**Comment by lornajane** [comment]
I think that would work, could you update the pull request?

File: specification/index.md (hunk 1 above)
Comment (karelhusa) [reply specification/index.md:20]: The 'description' is missing there. Can be one of:
- 'description' and 'example' (for simplicity)
- 'description', 'example' and 'examples' (for accuracy)

**Review by handrews — APPROVED** [review]

**Review by lornajane — APPROVED** [review]
Thanks!

**Pull request merged**